### PR TITLE
coreos-overlay: Add arm64 keywords

### PR DESCRIPTION
**Pull request by glevand**
Add the arm64 keyword to any ebuilds that need it.

### CODE REVIEW

File: app-emulation/open-vm-tools/open-vm-tools-9.10.0.ebuild [hunk 1]
```diff
@@ -17,7 +17,7 @@ EGIT_SOURCEDIR="${WORKDIR}"
 
 LICENSE="LGPL-2"
 SLOT="0"
-KEYWORDS="amd64 ~x86"
+KEYWORDS="amd64 arm64 ~x86"
```
Comment (marineam) [inline app-emulation/open-vm-tools/open-vm-tools-9.10.0.ebuild]: open-vm-tools is only applicable to VMware so I doubt this makes sense. Probably should fix by using `arm64? (  app-emulation/open-vm-tools )` or similar in board-packages.

File: app-emulation/xenserver-pv-version/xenserver-pv-version-6.2.0.ebuild [hunk 2]
```diff
@@ -12,7 +12,7 @@ SRC_URI=""
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="amd64"
+KEYWORDS="amd64 arm64"
```
Comment (marineam) [inline app-emulation/xenserver-pv-version/xenserver-pv-version-6.2.0.ebuild]: Only applicable to XenServer deployments, unlikely to be useful on ARM I assume. Add a conditional depend to the coreos ebuild instead.

**Comment by glevand** [comment]
@marineam I removed the two patches for open-vm-tools and xenserver-pv-version from this series and will include new patches with your recommended changes in another PR.

**Comment by marineam** [comment]
LGTM

**Pull request merged**